presto-hive/src/main/java/com/facebook/presto/hive/metastore/file/TableMetadata.java

```diff
@@ -279,6 +279,7 @@ public Table toTable(String databaseName, String tableName, String location)
                 partitionColumns,
                 parameters,
                 viewOriginalText,
-                viewExpandedText);
+                viewExpandedText,
+                false);
```

Comment:
I'm not familiar with `FileHiveMetastore`. Why is it always `false`?

Reply:
Now i use tableType to mark a table as a temporary table.